scripts/findUnused.ts

```diff
@@ -1,32 +1,140 @@
 import { relative } from "node:path"
-import { BindingNamedNode, Project } from "ts-morph"
+import { fromHere } from "@re-/node"
+import { BindingNamedNode, Node, Project, ts } from "ts-morph"
 
 const project = new Project({
-    tsConfigFilePath: "tsconfig.references.json"
+    tsConfigFilePath: fromHere("..", "tsconfig.references.json")
 })
 
 const unused: Record<string, string[]> = {}
+const snippetFileRegex = /snippets\/\w+/
```

Comment:
Since the purpose of this is to ignore snippet files, let's preemptively convert it to a `RegExp[]` called something like `ignorePaths` so if we need to add new ignore paths in the future, it will be obvious how that should be done.